beacon-chain/blockchain/chain_info.go

```diff
@@ -544,12 +545,6 @@ func (s *Service) recoverStateSummary(ctx context.Context, blockRoot [32]byte) (
 	return nil, errBlockDoesNotExist
 }
 
-// PayloadBeingSynced returns whether the payload for the block with the given
-// root is currently being synced and what is the withheld status in the payload
-func (s *Service) PayloadBeingSynced(root [32]byte) (primitives.PTCStatus, bool) {
```

Comment:
No longer used by external services. It is better not to have to export it if we can